sysdig/sysdig_clients.go

```diff
@@ -12,6 +12,9 @@ import (
 )
 
 type SysdigClients interface {
+	GetSecureEndpoint() (string, error)
```

Comment:
dunno if this is the best place from where to fetch the two attributes.
accept any other suggestion